iks-life-preserver/README.md

```diff
@@ -0,0 +1,67 @@
+# iks-life-preserver
+
+## What is this?
+Life Preserver is a simple project designed to be used with the 
+IBM Cloud Schematics service to make the process of inviting 
+someone to your IBM Cloud account for the purpose of helping you 
+troubleshoot issues with your IBM Cloud Kubernetes Service Clusters
+ (including OpenShift on IBM Cloud clusters).
+
+## How it works
+Follow the steps below to invite a user to your account and grant them: 
+
+* A `Viewer` IAM platform policy that allows the user to see only Kubernetes and OpenShift clusters
+* A `Reader` IAM service policy that allows the user to have **view-only** access to the **specific cluster you grant them access to**. 
+
+> Note: In terms of Kubernetes RBAC, this user will not have access to any Secrets or RBAC resources inside the cluster.
+
+
+### Step 1
+
+ Login to the IBM Cloud web console at https://cloud.ibm.com.
+
+### Step 2 
+
+Browse to the IBM Cloud Schematics offering at https://cloud.ibm.com/schematics
+
+### Step 3
+
+Click `Create workspace` and give your new workspace a relevant name and pick a location and click `Create`. 
```

Comment:
```suggestion
Click **Create Workspace**. Enter a name, select a location, and click **Create**.
```

Reply:
does this workspace need to be in the same RG as the cluster?

Reply:
It does not.